docs/01_application_environment_setup.md

```
@@ -0,0 +1,152 @@
+  const user = await authClient.getLoggedInUser();
+  const config = await configRepo.fetchPublicConfig();
+  const i18nLanguage = user?.settings?.language || 'en';
+
+  i18n.changeLanguage(i18nLanguage);
+
+  return { user, config };
+}
+```
+
+🧾 What’s Happening:
+- `authClient.getLoggedInUser()`: Asks the server, “who’s currently logged in?”
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)